cypress/e2e/po/components/tabbed.po.ts

```diff
@@ -10,7 +10,7 @@ export default class TabbedPo extends ComponentPo {
   }
 
   clickTabWithSelector(selector: string) {
-    return this.self().get(`${ selector }`).click();
+    return this.self().find(`${ selector }`).click();
```

Comment:
Switching from `get` to `find` ensures we click the right tab when the page has multiple tab groups with the same tabs